docs/update_and_migration/from_4.5/update_from_4.5.md

```diff
@@ -514,10 +514,38 @@ composer config extra.runtime.error_handler "\\Ibexa\\Contracts\\Core\\MVC\\Symf
 composer dump-autoload
 ```
 
+### Ibexa Cloud
+
+Update Platform.sh configuration for PHP 8.3.
+
+Generate new configuration with the following command:
+
+```bash
+composer ibexa:setup --platformsh
+```
+
+Review the changes applied to `.platform.app.yaml` and `.platform/`,
+merge with your custom settings if needed, and commit them to Git.
+
```

Comment:
Not sure it will still be needed there when 4.6.11 will be out. To do it only once at the last moment should be enough.

```suggestion
```